web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataService.kt

```diff
@@ -58,11 +58,17 @@ class PersondataServiceImpl(
         val tilganger = PersondataResult
             .runCatching(InformasjonElement.VEILEDER_ROLLER) { hentTilganger() }
             .getOrElse(PersondataService.Tilganger(kode6 = false, kode7 = false))
+        val kontaktinformasjonTredjepartsperson = PersondataResult.runCatching(InformasjonElement.DKIF) {
+            persondata
+                .findKontaktinformasjonTredjepartspersoner()
+                .map { dkif.hentDigitalKontaktinformasjon(it) }
+                .associateBy { it.personident }
+        }.getOrNull()
```

Comment:
Her er `it.personident` en `String?`, mens jeg da egentlig aller helst ville brukt identen(e) som man får fra `findKontaktinformasjonTredjepartspersoner()` som er `String`.

Reply:
Aha, ja jeg kunne jo bare kjørt `it to` inni der ja 🤔 Supert!